applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -193,6 +194,42 @@ static struct bt_audio_broadcast_sink_cb broadcast_sink_cbs = { .scan_recv = sca
 								.base_recv = base_recv_cb,
 								.syncable = syncable_cb };
 
+static void initialize(le_audio_receive_cb recv_cb)
+{
+	static bool initialized;
+
+	if (!initialized) {
+		receive_cb = recv_cb;
+
+		bt_audio_broadcast_sink_register_cb(&broadcast_sink_cbs);
+
+		for (size_t i = 0U; i < ARRAY_SIZE(streams); i++) {
+			streams[i].ops = &stream_ops;
+		}
+
+		initialized = true;
+	}
+}
+
+static void reset(void)
```

Comment:
Probably we can call this API "broadcast_sink_cleanup" or something.

Reply:
Agree